hoot-core-test/src/test/cpp/hoot/core/io/OsmApiWriterTest.cpp

```diff
@@ -205,7 +205,7 @@ class OsmApiWriterTest : public HootTestFixture
       CPPUNIT_ASSERT(writer.containsFailed());
       HOOT_STR_EQUALS(
         "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
-        "<osmChange version=\"0.6\" generator=\"Hootenanny\">\n"
+        "<osmChange version=\"0.6\" generator=\"hootenanny\">\n"
```

Comment:
Since this test wasn't enabled (and still isn't because it relies on an OSM API being available) it didn't get the updated lowercase hootenanny as the generator when I standardized them a while ago.